libyul/optimiser/SSATransform.cpp

```diff
@@ -87,14 +88,16 @@ void SSATransform::operator()(Block& _block)
 					visit(*varDecl.value);
 				if (varDecl.variables.size() != 1 || !m_variablesToReplace.count(varDecl.variables.front().name))
 					return {};
+				vector<Statement> v;
 				// Replace "let a := v" by "let a_1 := v  let a := v"
```

Comment:
Unrelated but can be fixed with this pr: I think it should be 
```suggestion
				// Replace "let a := v" by "let a_1 := v  let a := a_1"
```

Reply:
Yep, that's right!